stubs/javax.crypto.astub

```diff
@@ -12,9 +12,9 @@ import com.amazon.checkerframework.cryptopolicy.qual.*;
 // https://docs.oracle.com/javase/8/docs/technotes/guides/security/StandardNames.html#Cipher
 // Algorithms are not case-sensitive.
 class Cipher {
-	static Cipher getInstance(@CryptoWhiteListed({"aes/gcm.*", "rsa/ecb.*"}) String algorithm);
+	static Cipher getInstance(@CryptoWhiteListed(value={"aes/gcm.*", "rsa/ecb.*"}) String algorithm);
```

Comment:
You shouldn't the `value=` part when that's the only field you're setting - it's implicit.

Reply:
unfortunately, you do. you can only drop it if the annotation has only a single field. But here, there are two fields and one is called value. Maybe it shouldn't be called value. But that's a separate discussion to be had.

Reply:
From the [JLS 9.7.3](https://docs.oracle.com/javase/specs/jls/se9/html/jls-9.html#jls-9.7.3):
"It is legal to use single-element annotations for annotation types with multiple elements, so long as one element is named value and all other elements have default values (§9.6.2)."

So you should give a default value to the warnings attribute so that what I'm suggesting will work, and then do that :)